livebook/parsers/google_sheets/parser.py

```diff
@@ -0,0 +1,121 @@
+import gspread
+from google.oauth2 import service_account
+from google.auth.transport.requests import AuthorizedSession
+from livebook.parsers import *
+
+# If modifying these scopes, delete the file token.pickle.
+scopes = ['https://www.googleapis.com/auth/spreadsheets.readonly']
+# The ID and range of the needed spreadsheet.
+spreadsheet_id = '1PNWun9wY9cFdBs4FbDcnWPh6FGWnvJ6pNluKc1n0cpo'
+# Get credentials from service account file
+creds = service_account.Credentials.from_service_account_file('livebook_client_credentials.json', scopes=scopes)
+# Create a client for gspread
+client = gspread.Client(auth=creds)
+# Create a session for it
+client.session = AuthorizedSession(creds)
+# Get the sheet
+sheet = client.open_by_key(spreadsheet_id)
+# We'll nearly always need the meta sheet
+meta_sheet = sheet.worksheet("livebook_meta")
+
+# TODO: Good idea?
+'''
+Module variables
+'''
+stats_end = 1
```

Comment:
We can create a class Parser, and move the stats_end as a field. In such a way we can avoid using unnecessary column_val variable.

Reply:
The idea was that the parser is basically a Singleton, so we can use it as a module, without creating classes. Therefore, we can use module variables and everything else without caring too much.